src/main/java/com/troubleshooting/model/ILogModel.java

```diff
@@ -0,0 +1,11 @@
+package com.troubleshooting.model;
+
+// Log interface for Conjur and DAP logs
```

Comment:
Starting an interface name with a capital I is a C# convention.
an interface should have only methods.